plugin/pkg/admission/podtolerationrestriction/admission.go

```diff
@@ -162,6 +164,21 @@ func (p *podTolerationsPlugin) Admit(a admission.Attributes) error {
 		}
 	}
 
+	if qoshelper.GetPodQOS(pod) != api.PodQOSBestEffort {
```

Comment:
hm... , do you means this admission controller need to be the latest one in apiserver's args?